RFC/src/RFC-0140_Syncing_and_seeding.md

```diff
@@ -153,10 +153,9 @@ When running in pruning mode, [Base Node]s MUST remove all spent outputs that ar
 [utxo]: Glossary.md#unspent-transaction-outputs
 [mimblewimble]: Glossary.md#mimblewimble
 [mempool]: Glossary.md#mempool
-[ValidationState]: Glossary.md#validationstate
 [BroadcastStrategy]: Glossary.md#broadcaststrategy
 [range proof]: Glossary.md#range-proof
 [SynchronizationStrategy]: Glossary.md#synchronisationstrategy
 [SynchronizationState]: Glossary.md#synchronisationstate
 [mining server]: Glossary.md#mining-server
-[cut-through]: RFC-0110_BaseNodes.md#Pruning-and-cut-through
\ No newline at end of file
+[cut-through]: RFC-0110_BaseNodes.md#Pruning-and-cut-through
```

Comment:
Should this not just be made `pruning`?

Reply:
Will refine this a bit to reflect that cut-through is still defined in our system (MW) but prevented